external/DoCIF

```diff
@@ -1 +1 @@
-Subproject commit e52def19124bec4c9b743a8f7276ef2b14f64b51
+Subproject commit 1dec20b1995d4735b7ec295a6ec4ae4d584ee187
```

Comment:
This was probably changed on accident.  Do `git checkout robojackets/master --external`, then commit to revert it back to what it was.